src/components/quickLaunch/QuickLaunch.js

```diff
@@ -0,0 +1,65 @@
+/**
+ *
+ * @author sriram
+ *
+ */
+import React from "react";
+import Chip from "@material-ui/core/Chip";
+import { withStyles } from "@material-ui/core/styles";
+import PropTypes from "prop-types";
+import PublicIcon from "@material-ui/icons/Public";
+import LockIcon from "@material-ui/icons/Lock";
+
+const styles1 = (theme) => ({
+    chip: {
+        margin: theme.spacing.unit,
+    },
+});
+
+function QuickLaunch(props) {
+    const {
+        label,
+        handleClick,
+        handleDelete,
+        isDefault,
+        isPublic,
+        classes,
+    } = props;
+    const color = isDefault || isPublic ? "primary" : "";
+    const icon = isPublic ? <PublicIcon /> : <LockIcon />;
+    if (color) {
+        return (
+            <Chip
```

Comment:
I'm not 100% up to speed on all that's going into Quick Launch / Badge.  As I understand it, there's the internal QLs within the DE/CyVerse.run and there's the external QLs (or more commonly, "badges").  I'm assuming these will be for the internal usage and what Mariah mocked up, with the CyVerse logo, will be used externally?

Reply:
✔️ . Correct. These are for showing QL within DE / cyverse.run. Badges can be created by copying the embed code that will be displayed to users in App Info Dialog.